modules/install-vault/install-vault

```diff
@@ -102,6 +102,8 @@ function two_way_symlink() {
 
 # Install steps are based on: http://stackoverflow.com/a/31576473/483528
 function install_supervisord_amazon_linux {
+  if [[ ! $(pip list |grep supervisor) ]]; then
```

Comment:
Couldn't `supervisor` be installed via tools other than `pip`? And won't this return false positives for _anything_ that has the word "supervisor" in the name? 

Seems like a better check might be for the presence of `supervisorctl` in the `PATH`.

Reply:
Also, to avoid wrapping this whole method in an `if`, consider inverting the check and doing an early return right at the top.

Reply:
Removed this check entirely.  This was something I did on my local instance because it simply timed out with pip (no internet connectivity).  So I had installed the package manually and just ran a check to bypass.

Reverted to the the prior code.